crates/ruff_linter/src/rules/ruff/rules/sequence_sorting.rs

```diff
@@ -6,16 +6,18 @@
 use std::borrow::Cow;
 use std::cmp::Ordering;
 
+use is_macro;
+use natord;
+
```

Comment:
We don't actually need these imports at all here. I didn't really understand properly how Rust imports worked when I wrote this file 😶

```suggestion
```

Reply:
How did you even find this in such a large diff :laughing: